tokens/connect/ledger.md

```diff
@@ -12,7 +12,9 @@ description: This guide walks you through how to use your Ledger hardware wallet
 
 Hardware wallets provide a safer way to store crypto funds because the private key (used for signing transactions) is stored offline. Ledger offers two hardware wallet solutions at the time of writing: Ledger Nano S and Ledger Nano X.
 
-With the release of the Moonriver app on Ledger Live, you can now use your Ledger device to sign transactions on Moonriver without having to deal with chain configurations. If you want to use your Ledger device with one of the other Moonbeam-based networks, you can use the Ethereum app on Ledger Live by setting the chain ID. 
+For Moonbeam and the Moonbase Alpha TestNet, you can use the Ethereum app on Ledger Live by setting the chain ID. For Moonbeam, the chain ID is 1284, and for Moonbase Alpha, it's 1287.
+
+For Moonriver, you can use the Moonriver app on Ledger Live, this way you do not have to worry about setting the chain ID. Please note that the Moonriver app can only be used for connecting to the Moonriver network, it will not work for Moonbeam or Moonbase Alpha.
```

Comment:
I would say that the Ethereum app works for Moonriver as well, but the added bonus of using the dedicated app is that ensures you are connected to the right network (or something like that)